spec/models/work_spec.rb

```diff
@@ -161,6 +176,54 @@
       work = create(:work, fandoms: ships)
       expect(work.crossover).to be_truthy
     end
+
+    context "when one tagged fandom has two unrelated meta tags" do
+      let(:meta1) { create(:canonical_fandom) }
+      let(:meta2) { create(:canonical_fandom) }
+      let(:fandom) { create(:canonical_fandom) }
+
+      before do
+        fandom.update_attribute(:meta_tag_string, "#{meta1.name},#{meta2.name}")
+      end
+
+      it "is not a crossover with the fandom's synonym" do
```

Comment:
This is similar to https://github.com/otwcode/otwarchive/blob/0a980356c82bad64cc248e7b5501106163a27864/spec/models/work_spec.rb#L112-L117

Reply:
Should I delete the existing one? I figured it would be good to keep around both, to make it easier to pinpoint the source of the error if this code ever needs to be modified again. (e.g. Is the issue synonyms alone, or synonyms of fandoms with two top-level meta tags?)

Reply:
Good point, the tests cover different cases. Let's keep both.